src/assets/languages/english.json

```diff
@@ -238,8 +240,38 @@
         "heading": "Prevention",
         "stopTheBreed": {
           "heading": "Eliminate all the mosquito breeding places by clicking over them ",
-          "alerts" : {
-            "title" : "Congratulations!! <br> You eliminated a "
+          "alerts": {
+            "title": "Congratulations!! <br> You eliminated a "
+          }
+        },
+        "stridesAgainst": {
+          "heading": "Stride Against Malaria",
+          "howtoplay": {
+            "heading": "Click over the man to help moving it. You should reach the final point."
```

Comment:
Why isn't this changed to the way I suggested?

Reply:
@Raxerz I guess I was not able to understand the intention. Done the changes. Are they correct?